Tarea 3

rodoart opened the pull request:
Esta es mi propuesta de tarea 3. Rodolfo.

rodoart committed:
Merge remote-tracking branch 'refs/remotes/lbenet/master'

rodoart committed:
tarea0

Aquí está mi pull request

rodoart committed:
Ya hice git add, git commit.

Espero tus modificaciones Uriel.

Gracias

UriAceves committed:
Agrego linea

rodoart committed:
Merge branch 'master' of https://github.com/rodoart/2017-1_TSFisComp-1

Mezclando archivos.

rodoart committed:
Merge pull request #1 from lbenet/master

Esto es una prueba

rodoart committed:
Merge pull request #2 from lbenet/master

Actualización online (Sin consola 2)

rodoart committed:
Merge pull request #3 from lbenet/master

Actualizacion 2

rodoart committed:
Create r

rodoart committed:
Subo mi tarea 3

Rodolfo.

rodoart committed:
Delete r

UriAceves commented:
> También se trata de un punto atractor, pero ahora no está en el 0.0 si no en un punto cercano a -4.0 

En eso estoy de acuerdo, pero veo que sólo lo verificaste por un lado, checa por el otro también para que tengan paz nuestras almas.

> Obsevamos que el 0.0 es un punto fijo atractivo, pero a cada iteración el valor se acerca más lentamente.

¿Estamos seguros? ¿Por los dos lados es atractivo? Sólo muéstralo gráficamente.

rodoart committed:
Correciones a la versión anterior.

UriAceves commented:
> Del lado derecho es repulsivo, las órbitas se alejan.

Justo al final, de hecho no. Es atractivo hasta que superas cierto punto. Igual y podrías comentar un poco de esto.

Cuando tengas esto listo hago el merge.

rodoart committed:
Corregida la conclusión final.

Me di cuenta de la dinámica real al agregar más puntos y agrandar el intervalo.

UriAceves commented:
Listo.

Hago el merge.
